front-end/src/reducers/rootReducer.js

```diff
@@ -1,15 +1,8 @@
+import { combineReducers } from 'redux'
 import loanReducer from './loanReducer'
-import productReducer from './productReducer'
+import loadingReducer from './loadingReducer'
+// import productReducer from './productReducer' for next task
```

Comment:
@JialinWang1 I'm pretty sure whoever is going to that, can do the import without this help :D

Reply:
that one is me